Cargo.toml

```diff
@@ -20,7 +20,7 @@ rpc = ["dep:relay_rpc"]
 
 [dependencies]
 relay_client = { path = "./relay_client", optional = true }
-relay_rpc = { path = "./relay_rpc", optional = true }
+relay_rpc = { path = "./relay_rpc", optional = true, features = ["cacao"] }
```

Comment:
Why do we want optional features to be on by default?

We should probably get rid of the root crate altogether, as it doesn't make sense to use it most of the time. It only makes configuring features harder.

Reply:
`just build` wasn't building this feature/module so I wasn't able to tes it and my IDE wasn't triggering on errors. So this seemed to fix that. I thought this affected CI as well, but looks like `cargo build --workspace --all-features` will check this (as CI is configured to).

Going to revert to avoid confusion as CI is already catching any potential issues.

Reply:
Not sure why we have the root crate like this, it doesn't seem like we use `walletconnect_sdk` anywhere. Maybe it's for the examples?